import-automation/progress-dashboard-rest/app/main.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Entry point of the API.
+When deployed on App Engine, gunicorn will serve the FLASK_APP variable,
+which is a Flask object.
```

Comment:
How does Flask fit into the services we are using?  Do we have a diagram?

Reply:
Flask is just a web API framework